supplier_app/forms.py

```diff
@@ -61,29 +118,33 @@ class Meta:
         }
 
 
-class TaxPayerCreateForm(ModelForm):
+class TaxPayerCreateForm(BasePrefixCreateForm):
     prefix = 'taxpayer_form'
 
-    def __init__(self, user=None, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        # self.fields['company'].queryset = self.get_user_companies(user)
-
-    def get_user_companies(self, user):
-        return Company.objects.filter(companyuserpermission__user=user)
-
     class Meta:
         model = TaxPayerArgentina
         exclude = ['taxpayer_state', 'workday_id', 'company', 'country']
         widgets = {
             'name': forms.TextInput(attrs={'class': 'form-control'}),
-            'razon_social': forms.TextInput(attrs={'class': 'form-control'}),
+            'business_name': forms.TextInput(attrs={'class': 'form-control'}),
```

Comment:
Could you explain me the difference between `name` and `business_name`?

Reply:
You have reason it is the same

Reply:
We should erase it? or are you considering erase it later?

Reply:
I removed it